### PR TITLE
Added note about SharePoint libraries

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-retention-policies.md [hunk 1]
```diff
@@ -176,7 +176,8 @@ When you choose the **SharePoint sites** location, the retention policy can reta
 
 Although the retention policy is applied at the site level, only documents have retention settings applied to them. Retention settings do not apply to the organizing structures that include libraries, lists, and folders within the site.
 
-When you specify your locations for SharePoint sites or OneDrive accounts, you don't need permissions to access the sites and no validation is done at the time you specify the URL on the **Edit locations** page. However, the SharePoint sites must be indexed and the sites that you specify are checked that they exist at the end of the wizard.
+When you specify your locations for SharePoint sites or OneDrive accounts, you don't need permissions to access the sites and no validation is done at the time you specify the URL on the **Edit locations** page. However, the SharePoint sites must be indexed and the sites that you specify are checked that they exist at the end of the wizard. 
+Documents in SharePoint Libraries that are set to not appear in search results have retention settings applied to them as well if site is set to be indexed.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-retention-policies.md]:
```suggestion
Documents in SharePoint libraries that are set to not appear in search results have retention settings applied to them as well if the site is set to be indexed. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@PramodBalusu Please implement the suggested changes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks.

**Comment by cabailey** [comment]
#hold-off  @yogkumgit  This doesn't read well, so need to find better wording.  Leave it with me.

**Comment by cabailey** [comment]
This is updated on the live site now, with rewording, so closing this.  Look for the new Note: "SharePoint sites must be indexed for the retention settings to be applied. However, if items in SharePoint document libraries are configured to not appear in search results, this configuration doesn't exclude the items from the retention settings."

**Pull request closed without merging**